cactus/tests/test_plugins.py

```diff
@@ -71,4 +76,143 @@ def test_call(self):
 
         #postBuild
         self.assertEqual(1, len(plugin.postBuild.calls))
-        self.assertEqual((self.site,), plugin.postBuild.calls[0]['args'])
\ No newline at end of file
+        self.assertEqual((self.site,), plugin.postBuild.calls[0]['args'])
+
+
+class TestPagecontextPlugin(unittest.TestCase):
```

Comment:
Can you move that to a `test_page_context.py` file  (e.g. we have `test_compat.py` for the compatp lugin, etc, etc)